torch_xla/csrc/ops/ops.cpp

```diff
@@ -882,26 +883,50 @@ NodePtr NanToNum(const Value& input, const Value& nan, const Value& posinf,
                    input.shape(), std::move(lower_fn));
 }
 
-NodePtr DynamicExpand(const Value& static_input, const Value& dynamic_target) {
-  auto lower_fn = [](const Node& node, LoweringContext* loctx) -> XlaOpVector {
+NodePtr DynamicExpand(const Value& input, const std::vector<xla::int64> static_size, const xla::int64 dynamic_size, const Value& dynamic_target) {
```

Comment:
probably 
```cpp
, const std::vector<xla::int64> static_size
```
was meant to be passed by ref? Also seems passed by value into shape_fn?